Formula/gist.rb

```diff
@@ -14,7 +14,7 @@ class Gist < Formula
     sha256 cellar: :any_skip_relocation, high_sierra:   "28f8947a2912459cc79536eed7cbc2af958c9282f31990741f2fd0f32fffa70a"
   end
 
-  depends_on "ruby" if MacOS.version <= :sierra
+  uses_from_macos "ruby", since: :sierra
```

Comment:
```suggestion
  uses_from_macos "ruby", since: :high_sierra
```
?

Reply:
Actually it's for versions before `Sierra`, so it should be `uses_from_macos "ruby", before: :high_sierra` ?

Reply:
Is `:before` a legal key in that hash table? If it is, it should be equivalent. If it isn't, we need `since: :high_sierra`.

Reply:
Actually, maybe it might not be equivalent, because `before: :sierra` could mean `MacOS.version < :sierra`.

Reply:
Read it as "uses system Ruby since High Sierra" (and brewed Ruby otherwise).

So:

```rb
uses_from_macos "ruby", since: :high_sierra
```

means

```rb
depends_on "ruby" if MacOS.version < :high_sierra
```

Reply:
Got it, thanks!